pkg/mic/mic.go

```diff
@@ -310,8 +310,15 @@ func (c *Client) Sync(exit <-chan struct{}) {
 			c.getListOfIdsToAssign(*addList, nodeMap)
 		}
 
+		var wg sync.WaitGroup
```

Comment:
Can you please add a comment near nodeMap make to indicate that the key can be now vmss or vm name.